NEWS.md

```diff
@@ -9,6 +9,7 @@
   interfaces with `obj_type_friendly()` from `compat-obj-type.R` via a
   C callable.
 
+* The `pkg_version_info()` function now allows `==` for package version comparison. ([#1469](https://github.com/r-lib/rlang/issues/1469), [@kryekuzhinieri](https://github.com/KryeKuzhinieri))
```

Comment:
The bullet should be under the development version header.

Reply:
Shall I add it under `1.0.5.9000 `or increment the development version to `1.0.6`?

Reply:
Just add the bullet under the very first header in the file, thanks!